server/routes/index.ts

```diff
@@ -28,6 +29,24 @@ export default function routes(services: Services): Router {
     get('/incident/:id', debugRoutes.incidentDetails)
     get('/report/:id', debugRoutes.reportDetails)
 
+    get('/prisoner/:prisonerNumber/photo.jpeg', async (req, res) => {
+      const { user } = res.locals
+      const { prisonerNumber } = req.params
+
+      const prisonApi = new PrisonApi(user.token)
+      const photoData = await prisonApi.getPhoto(prisonerNumber)
+
+      const oneDay = 86400 as const
+      res.setHeader('Cache-Control', `private, max-age=${oneDay}`)
+      res.setHeader('Content-Type', 'image/jpeg')
+
+      if (!photoData) {
+        res.sendFile('prisoner.jpeg', { root: `${services.applicationInfo.packageJsonPath}/assets/images` })
```

Comment:
Is this file missing?

Reply:
No you can see within the repo there is a `assets/images` folder from the top level

Reply:
Oh yeah the placeholder prisoner photo is missing. I will bring that in now